apps/melding-form/src/app/(general)/Home.test.tsx

```diff
@@ -35,4 +36,37 @@ describe('Page', () => {
 
     expect(screen.queryByText('Test error message')).toBeInTheDocument()
   })
+
+  it('should fetch defaultValues from localStorage', async () => {
+    const store: Record<string, string> = { textArea1: 'Test input textarea' }
+
+    global.localStorage = {
+      getItem: vi.fn((key: string) => JSON.stringify(store[key]) ?? null),
+    } as unknown as Storage
+
+    render(<Home formData={[mockTextAreaComponent]} />)
+
+    const textAreaInput = screen.getByRole('textbox', { name: 'What is it about? (niet verplicht)' })
+
+    expect(textAreaInput).toHaveValue('Test input textarea')
+  })
+
+  it('should handle onChange and set value in localStorage', async () => {
+    const store: Record<string, string> = { textArea1: '' }
+
+    global.localStorage = {
+      getItem: vi.fn((key: string) => JSON.stringify(store[key]) ?? null),
+      setItem: vi.fn((key: string, value: string) => {
+        store[key] = value
+      }),
+    } as unknown as Storage
+
+    render(<Home formData={[mockTextAreaComponent]} />)
+
+    const textAreaInput = screen.getByRole('textbox', { name: 'What is it about? (niet verplicht)' })
+
+    await userEvent.type(textAreaInput, 'Foo bar')
```

Comment:
[Testing library recommends](https://testing-library.com/docs/user-event/setup) setting up user events slightly differently, see [the TextArea tests](https://github.com/Amsterdam/meldingen-frontend/blob/main/libs/form-renderer/src/components/TextArea/TextArea.test.tsx#L41) for example. Same thing on other pages.